common.mk

```diff
@@ -141,8 +139,10 @@ endif
 .PHONY: all clean quick
 
 quick: $(DEFAULT_BIN)
+	@if test -e $(DEFAULT_BIN); then $(ECHO) "Build done"; fi;
```

Comment:
Why was this added?

Reply:
To show the command has executed.
If there's nothing to do in command `quick`, the compiler will echo `there is nothing to do in 'quick'`
But in my new version, there are two commands in `quick` and the message wouldn't display anymore.

I think you have a better idea.